README.md

```diff
@@ -1 +1,287 @@
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com: https://github.com/LethaboMot/built-portfilo.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@LethaboMot](https://github.com/LethaboMot)
+- LinkedIn: [LethaboMotloung](https://www.linkedin.com/in/lethabo-motloung-a8157624b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly make the following changes to your README.md file:

- Remove the Microverse logo. If you have your logo, you can use it, otherwise, I would advise you to remove the logo. 👍
- In the **_Tech stack section_** of the README, kindly remove the technologies listed in the `Client`, `Server` and `Database` subsections. Those sections contain technologies that were not used in this project. Kindly remove the `Server` and `Database` subsections and replace those in the `Client` subsections with HTML and CSS. 👍 😎 
- In the **_Key Features section_**, kindly add 1-3 things that you accomplished in this project. I would also advise you to get used to adding these features because you would do that for every project in Microverse and possibly in the professional world. 👍
- If you are not required to deploy the application, you can remove the **_Live Demo section_** of the README.md file. If you are required to deploy, then the link to the deployed project should be pasted under this section. 👍
- In the **_Getting Started Section_**, kindly do the following for the subsections:
  -  In the **_Prerequisite subsection_**, kindly add packages or applications that would be needed to start this application on the local machine. You can include NPM, a browser, a code editor, git and any other thing you can think of. 👍  
  - In the _Setup subsection_, kindly include ways to clone this project and change the directory to the directory of the project. See the example below as a guide:

```sh
  cd my-folder
  git clone git@github.com:myaccount/my-project.git
  cd my-project
```
  - In the **_Install subsection_**, you can indicate ways to install the linters locally and also to run them locally. 👍 😎
  - The _**Usage**_ subsection should contain ways to run the project in the browser if while the Run tests subsections should contain the commands for running the tests written in the project. 👍 😎 
- The **_Future Features_** section of the README.md file. The Future Features section should contain 1-3 features that can be implemented in the future. Most times, this section acts as a reference to you and future collaborators whenever you want to improve your projects in the future 💻 😎 
- The **_Show Your Support_** Section should communicate to people ways they can help advance the project or give a star to the project if they like it 👍 😎 
- The _**Acknowledgement**_ section should contain information, links and appreciation for those who spurred or contributed to the success of the project 💻 
- The _**FAQ**_ section is the only optional section and it should contain frequently asked questions about the project and answers to those questions 😎 👍